mempool/src/shared_mempool/network.rs

```diff
@@ -63,24 +60,6 @@ pub enum MempoolSyncMsg {
     },
 }
 
-/// The interface from Network to Mempool layer.
-///
-/// `MempoolNetworkEvents` is a `Stream` of `PeerManagerNotification` where the
-/// raw `Bytes` direct-send and rpc messages are deserialized into
-/// `MempoolMessage` types. `MempoolNetworkEvents` is a thin wrapper around an
-/// `channel::Receiver<PeerManagerNotification>`.
-pub type MempoolNetworkEvents = NetworkEvents<MempoolSyncMsg>;
-
-/// Returns a network application config for the mempool client and service
-pub fn mempool_network_config(max_broadcasts_per_peer: usize) -> NetworkApplicationConfig {
-    NetworkApplicationConfig::client_and_service(
-        [ProtocolId::MempoolDirectSend],
-        aptos_channel::Config::new(max_broadcasts_per_peer)
-            .queue_style(QueueStyle::KLAST)
-            .counters(&counters::PENDING_MEMPOOL_NETWORK_EVENTS),
-    )
-}
-
```

Comment:
so we move more data away from the application that uses networking and more into the node... hmm...

Reply:
See my response below 😸